opm/output/Cells.hpp

```diff
@@ -32,6 +32,10 @@ class SimulationDataContainer;
 namespace data {
 
     struct CellData {
+        CellData(std::string name, UnitSystem::measure dim, std::vector<double> data)
```

Comment:
Since this is a simple struct you can initialize it using `CellData{ name, dim, data }` even without this constructor, isn't that right?